OrbitGl/TextBox.h

```diff
@@ -94,8 +97,9 @@ class TextBox
     Color        m_Color;
     Timer        m_Timer;
     int          m_MainFrameCounter;
-    bool		 m_Selected;
+    bool         m_Selected;
     float        m_TextY;
+    size_t       m_ElapsedTimeTextLength;
```

Comment:
Uninitialized variable.

Reply:
RIP in peace 💀 

Fixed the constructors, will push updated PR in a bit

Reply:
Great!  Thanks Rob!